client/src/test/java/org/spine3/base/CommandsShould.java

```diff
@@ -81,10 +84,13 @@ public void have_private_ctor() {
 
     @Test
     public void pass_null_tolerance_test() {
-        NullToleranceTest.newBuilder()
-                         .setClass(Commands.class)
-                         .build()
-                         .check();
+        final boolean passed = NullToleranceTest.newBuilder()
+                                                .setClass(Commands.class)
+                                                .addDefaultValue(DEFAULT_FILE_DESCRIPTOR)
+                                                .addDefaultValue(Timestamps.getCurrentTime()) // to fulfil custom validation
+                                                .build()
```

Comment:
Usage lookup shows that we have 3 cases of not calling `check()` after `build()`, but only 2 of them are in `NullToleranceTestShould`.